include/alpaka/vec/Vec.hpp

```diff
@@ -511,8 +512,7 @@ namespace alpaka
         }
 
     private:
-        // 16 Byte alignment for usage inside of CUDA kernels.
-        alignas(16u) TSize m_data[TDim::value];
+        alignas(ALPAKA_OPTIMAL_ALIGNMENT_SIZE(TSize[TDim::value])) TSize m_data[TDim::value];
```

Comment:
Thank you for investigating this!
I will remove all the `alignas` for now because the Version 1 within the vector does not have any influence (because this is the default alignment) and `WorkDivMembers` is not currently used in the CUDA GPU implementation.